test/applications_integration_test.go

```diff
@@ -90,17 +140,11 @@ func iBuildAnApplicationTransaction(
 	}
 
 	if approvalProgram != "" {
-		approvalP, err = ioutil.ReadFile("features/resources/" + approvalProgram)
-		if err != nil {
-			return err
-		}
+		approvalP, err = readTealProgram(approvalProgram)
```

Comment:
nit: do we want to abort here if this trigger an error?

Reply:
My mistake, I did not mean to get rid of the if statement